frontend/packages/dev-console/src/utils/resource-label-utils.ts

```diff
@@ -14,11 +14,13 @@ export const getAppLabels = (
   };
 };
 
-export const getAppAnnotations = (gitURL: string, gitRef: string) => {
-  const ref = gitRef || 'master';
+export const getAppAnnotations = (gitURL: string, gitRef: string, cheURL: string) => {
   return {
     'app.openshift.io/vcs-uri': gitURL,
-    'app.openshift.io/vcs-ref': ref,
+    'app.openshift.io/vcs-ref': gitRef || 'master',
+    ...(cheURL && {
+      'app.openshift.io/edit-url': `${cheURL}/f?url=${gitURL}&policys.create=peruser`,
```

Comment:
policys -> policies

Reply:
Updated. `policys` is what you mentioned in your earlier comment.

Reply:
I'm sorry. I was wrong. ```policies``` is the correct  query name